code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -55,6 +59,21 @@
 			playsound(src, 'sound/effects/mousesqueek.ogg', 100, 1)
 	..()
 
+/mob/living/simple_animal/mouse/handle_automated_action()
+	if(prob(chew_probability))

```

Comment:
So 1 out of every 100 it checks the tile its on to see if there is a wire then knaws on it if there is.
I would prefer it to have a second prob of 10 for chewing on the wire otherwise it just cleans itself.